docs/DynamicDNS.md

```diff
@@ -36,18 +32,18 @@ First, create an account on a Dynamic DNS provider, the most popular providers a
 * [dyndns](https://dyn.com/) (not free)
 * [google](https://domains.google.com/) (not free)
 
-Once you created an account, you can create a free domain name through their website.
+Once you've created an account, you can create a free domain name through their website.
 
-### Step 2: Configure Dynamic DNS on BTCPayServer
+### Step 2: Configure Dynamic DNS on BTCPay Server
 
-You need to be administrator of your instance, go to Server Settings / Services / Dynamic DNS.
+You need to be administrator of your instance.
+Go to Server Settings / Services / Dynamic DNS.
```

Comment:
Normally the docs format nav like this:
Server Settings > Services > Dynamic DNS